common/build/eslint-config-fluid/strict-ts.json

```diff
@@ -0,0 +1,1442 @@
+        "arrow-body-style": [
+            "off"
+        ],
+        "arrow-parens": [
+            "error",
+            "always"
+        ],
+        "brace-style": [
+            "off"
+        ],
+        "camelcase": [
+            "off"
+        ],
+        "capitalized-comments": [
+            "off"
+        ],
+        "comma-dangle": [
+            "off"
+        ],
+        "comma-spacing": [
+            "off"
+        ],
+        "complexity": [
+            "off"
+        ],
+        "constructor-super": [
+            "error"
+        ],
+        "curly": [
+            "error"
+        ],
+        "default-case": [
+            "error"
+        ],
+        "dot-notation": [
+            "off",
+            {
+                "allowKeywords": true,
+                "allowPattern": "^_"
+            }
+        ],
+        "editorconfig/charset": [
+            "error"
+        ],
+        "editorconfig/eol-last": [
+            "error"
+        ],
+        "editorconfig/indent": [
+            "off"
+        ],
+        "editorconfig/linebreak-style": [
+            "error"
+        ],
+        "editorconfig/no-trailing-spaces": [
+            "error"
+        ],
+        "eol-last": [
+            "off"
+        ],
+        "eqeqeq": [
+            "error",
+            "smart"
+        ],
+        "eslint-comments/disable-enable-pair": [
+            "error",
+            {
+                "allowWholeFile": true
+            }
+        ],
+        "eslint-comments/no-aggregating-enable": [
+            "error"
+        ],
+        "eslint-comments/no-duplicate-disable": [
+            "error"
+        ],
+        "eslint-comments/no-unlimited-disable": [
+            "error"
+        ],
+        "eslint-comments/no-unused-enable": [
+            "error"
+        ],
+        "for-direction": [
+            "warn"
+        ],
+        "func-call-spacing": [
+            "off"
+        ],
+        "getter-return": [
+            "off"
+        ],
+        "guard-for-in": [
+            "error"
+        ],
+        "id-match": [
+            "error"
+        ],
+        "import/default": [
+            2
+        ],
+        "import/export": [
+            2
+        ],
+        "import/named": [
+            "off"
+        ],
+        "import/namespace": [
+            2
+        ],
+        "import/no-default-export": [
+            "error"
+        ],
+        "import/no-deprecated": [
+            "off"
+        ],
+        "import/no-duplicates": [
+            1
+        ],
+        "import/no-extraneous-dependencies": [
+            "error",
+            {
+                "devDependencies": [
+                    "**/*.spec.ts",
+                    "src/test/**"
+                ]
+            }
+        ],
+        "import/no-internal-modules": [
+            "error"
+        ],
+        "import/no-named-as-default": [
+            1
+        ],
+        "import/no-named-as-default-member": [
+            1
+        ],
```

Comment:
Disable this per <https://typescript-eslint.io/docs/linting/troubleshooting/#eslint-plugin-import>